modules/storage/mysql-single-server/test/dependencies.test.bicep

```diff
@@ -0,0 +1,71 @@
+param location string
+param name string
+param prefix string
+
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2022-05-01' = {
+  name: '${name}-${prefix}-vnet'
+  location: location
+  properties: {
+    addressSpace: {
+      addressPrefixes: [
+        '10.0.0.0/16'
+      ]
+    }
+    subnets: [
+      {
+        name: '${name}-subnet-0'
+        properties: {
+          addressPrefix: '10.0.0.0/24'
+          privateEndpointNetworkPolicies: 'Disabled'
```

Comment:
will this be 'Disabled' for all usecases or should it be a parameter? (same for the other)